torch/csrc/jit/codegen/cuda/scheduler.cpp

```diff
@@ -415,15 +415,18 @@ c10::optional<ReductionParams> scheduleReduction(
   ReductionParams rparams = reductionHeuristic(
       red_elems.value(), red_outputs.value(), red_on_fastest_dim);
 
+  constexpr int kLoopUnrollSplit = 4;
+
   // Heuristic Definition
   if (rparams.fastest_dim_) {
     // Do multiple reductions per block
     if (rparams.mul_reds_per_blk_) {
-      // Unroll a certain number of rFactored elements
-      red_tv->split(1, 4);
+      // Reduction Splits
       red_tv->split(1, rparams.block_dim_x_);
-      // Unroll a certain number of rFactored elements
-      // Split Grid dimension to get multiple reds per block
+      red_tv->split(1, kLoopUnrollSplit);
+      red_tv->reorder({{-1, -2}, {-2, -1}});
```

Comment:
a comment would help here to explain the logic behind the reorder

Reply:
Can you increase the computeAt point because of the reorder?